main.py

```diff
@@ -376,6 +384,11 @@ def main():
         dest="generate_config", nargs="?" ,const="auto", type=str,
         required=False)
 
+    parser.add_argument('-D', '--debug', help='This will run any processes ' \
```

Comment:
I guess I'm not entirely sure the purpose of this new argument. What is this meant for and was this apart of the original card (I can't recall, I'll be double checking here in a moment)?

Reply:
No, it's not a part of the original card. I can remove it, if you want to, but I added it so that we could call `python main.py -D` to debug/test instead of having an untracked test.py file. This way, we can easily set up a test (or multiple tests) that can easily be ran from the other's end to verify that the work has been correctly implemented and/or so that we can possibly see what cases the other has missed.

If you wanted to, we could add some properties and/or protections to this method so that it doesn't show up in production, but I think the idea is solid.

Reply:
For instance, I could have added the test cases for the temp monitor here and we might have caught the missing scenario that caused the crash a couple of days ago.

Reply:
Ah, fair enough. Yeah, I was a little confused of its purpose but that makes sense. Yeah, I'm alright with keeping it. But I do agree that we should put some protections in it so it is not available in production.